File: Python/buzzer_test.py
import RPi.GPIO as GPIO

INFRARED = 7 #BCM.7 (wPi. 11, Physical.26)
BUZZER = 17 #BCM.17 (wPi. 0, Physical.11)
LED = 18 #BCM.18 (wPi. 1, Physical.12)

GPIO.setmode(GPIO.BCM)
GPIO.setwarnings(False)
GPIO.setup(INFRARED, GPIO.IN)
GPIO.setup(BUZZER, GPIO.OUT)
GPIO.setup(LED, GPIO.OUT)

try:
    while True:
        if GPIO.input(INFRARED) == 1:
            GPIO.output(BUZZER, True)
            GPIO.output(LED, True)
        else :
            GPIO.output(BUZZER, False)
            GPIO.output(LED, False)
except:
    GPIO.cleanup()
    print("end")

File: Csource/buzzer_test.c
#include <wiringPi.h>
#include <stdio.h>

#define INFRARED 11 // wPi. 11(BCM. 7, Physical.26)
#define BUZZER 0 // wPi. 0(BCM. 17, Physical.11)
#define LED 1 // wPi. 1(BCM. 18, Physical.12)

int main (void) {

    if (wiringPiSetup() == -1) { return 1; }


    pinMode(INFRARED, INPUT);

    pinMode(BUZZER, OUTPUT);
    pinMode(LED, OUTPUT);

    for (;;) {


        if (digitalRead(INFRARED) == HIGH) {

            digitalWrite(BUZZER, HIGH);
            digitalWrite(LED, HIGH);
        } else {

            digitalWrite(BUZZER, LOW);
            digitalWrite(LED, LOW);
        }

    }

    return 0;
}
